pkgs/development/python-modules/flashtext/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "flashtext";
+  version = "2.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1kq5idfp9skqkjdcld40igxn2yqjly8jpmxawkp0skwxw29jpgm1";
+  };
+
+  # json files that tests look for don't exist in the pypi dist
+  doCheck = false;
```

Comment:
@aanderse Please add a pythonImportsCheck. Also can we fetch from github to activate tests in addition to that?